CHANGELOG.md

```diff
@@ -4,7 +4,33 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [2.6] - Piellorieppe
+## [2.6.1](https://github.com/nf-core/sarek/releases/tag/2.6.1) - Gådokgaskatjåhkkå
+
+Gådokgaskatjåhkkå is the highest peak in the Piellorieppe massif.
+
+### Added
+
+### Changed
+
+- [#208](https://github.com/nf-core/sarek/pull/208) - Merge changes from the release PR
+- [#208](https://github.com/nf-core/sarek/pull/208) - Bump version to `3.0dev`
```

Comment:
2.6.1, not 3.0..

Reply:
Actually that PR did bump to 3.0dev. I'm just changing it out to 2.6.1 in this PR.